app/src/main/java/comcmput301f16t01/github/carrier/RiderList.java

```diff
@@ -1,22 +0,0 @@
-package comcmput301f16t01.github.carrier;
-
-import java.util.ArrayList;
-
-import comcmput301f16t01.github.carrier.Users.User;
-
-/**
- * TODO: Remove/delete class when nothing references RiderList
```

Comment:
no no. the classes are bigger than the methods. I dont know, just a thought. I kind of forgot that we were getting rid of these so it is a nice reminder? I dont know